### DISCUSSION TITLE
답장으로 챗을 가리거나 유저를 강퇴하는 방법

### DISCUSSION THREAD

**Discussion by ghost** [storycraft/node-kakao#275] [Q&A] [answered]
<font class="papago-parent"><font class="papago-source" style="display:none;">3버전 입니다...</font>3버전 입니다...</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">답장으로 메세지를 가리거나 답장한 유저에 아이디를 어떻게 가져와요?</font>답장으로 메세지를 가리거나 답장한 유저에 아이디를 어떻게 가져와요?</font>

**Comment by raravel** [comment, marked as answer]
이벤트를 받았을 때, 채팅이 답장형식인지 확인하는 법
```js
if ( chat.Type === ChatType.Reply ) {
   ...
}
```

답장이 가리키는 채팅을 삭제하는 법
```js
chat.Channel.hideChatId(chat.Reply.SourceLogId)
```

**Reply by ghost** [reply]
감사합니다

[a comment by an automated account is not shown]

**Reply by ghost** [reply]
감사합니다
